includes/ting_search_autocomplete.admin.inc

```diff
@@ -53,6 +53,7 @@ function ting_search_autocomplete_admin() {
     '#title' => t('Facet Index', array(), array('context' => 'ting_search_autocomplete')),
     '#size' => 25,
     '#default_value' => $default_settings['facetIndex'],
+    '#required' => TRUE,
```

Comment:
just so - the facetspell method has index and facetIndex as required parameters. See also: [http://testbroend.dbc.dk:7080/suggest/#facetspell]